Source/ReadHDF5RhoYeT.cpp

```diff
@@ -0,0 +1,182 @@
+#include <iostream>
+
+#include <AMReX_GpuAllocators.H>
+
+#define H5_USE_16_API 1
+#include "hdf5.h"
+
+#include "ReadHDF5RhoYeT.H"
+
+// mini NoMPI
+#define HAVE_CAPABILITY_MPI //FIXME: This should be defined only when USE_MPI = TRUE

```

Comment:
This should be changed such that code compiles with and without MPI. Have a look at the ReadEoSTable.cpp.